src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
+
+    void draw(){
+
+    }
```

Comment:
why do we create method draw() in the Figure class? 
What will happen if I want to draw some person? Should I extend Person from Figure?